### DISCUSSION TITLE
Issue getting the CLI working

### DISCUSSION THREAD

**Discussion by rmerzbac** [deta/space-docs#51] [General]
Hi! I'm trying to install the CLI, and I'm not able to get it working. After running "curl -fsSL https://get.deta.dev/space-cli.sh | sh" and restarting the terminal, I get the error "-bash: space: command not found". It does appear that a file named "space" was installed at /Users/[My account]/.detaspace/bin/space.zip. What are the next steps I should take to the debug the issue? Thanks!

**Comment by sudo-subg** [comment]
hey @rmerzbac, check the PATH environment variable to make sure the directory where the CLI is installed is included in it (something like `/home/<your_username>/.detaspace/bin`). You can use `echo $PATH` command to see the current PATH variable.
If it's not already there, you can add it yourself by opening the `.bashrc` file in the home directory and adding the following line: `export PATH="/home/<your_username>/.detaspace/bin:$PATH"`.

**Reply by rmerzbac** [reply]
That doesn't seem to be working. "export PATH="/Users/<your_username>/.detaspace/bin:$PATH" was in .bashrc already, but swapping out "home" in place of "Users" didn't seem to change anything. I should mention that I'm on Mac, not Linux. Any other ideas? Thanks for your help!

**Reply by andres-brun** [reply]
Hello @rmerzbac,

You only need to add it to your PATH environment variable. This is the command: export PATH=$PATH:/Users/your_user/.deta/bin

**Reply by sudo-subg** [reply]
> This is the command: export PATH=$PATH:/Users/your_user/.deta/bin

it will be export PATH=$PATH:/Users/your_user/.detaspace/bin

**Reply by ADv0rnik** [reply]
Hi, @rmerzbac 

Don't forget to reboot your terminal after that. It's an important step to make things moving. This works both for space-cli and deta-cli. Just run `space --help` or `deta --help`

**Comment by lete114** [comment]
If you are a web front-end developer and have nodejs on your computer, you can try this `npm install deta-space-cli -g`

**Comment by jokikim** [comment]
I seem to be getting the same error. I also see it in .bashrc, but neither the "space" nor "deta" commands are found.

**Reply by sudo-subg** [reply]
hey @jokikim, check in the `.zshrc` file, if it's not already there, add `export PATH=$PATH:/Users/<your_user_name>/.detaspace/bin`, and then it should work.
